### PR TITLE
Add pagination

### PR DESCRIPTION
**Pull request by superNazo**
[Link to JIRA ticket](https://ssu-jira.softserveinc.com/secure/RapidBoard.jspa?rapidView=484&view=planning&selectedIssue=LR-184)

### Author.
Nazarii Ivankiv

- [x] I have added link to this PR to the JIRA
- [x] I have logged time in JIRA.

### Reviewers.

- [ ] @YUzhva 
- [ ] @anatoliiibrahimov 
- [ ] @vovka

### Issue summary.

Added Pagination to Quizzes list page

### Testing approach.

Manual testing

### CODE REVIEW

File: vendor/assets/javascripts/dirPagination.js [hunk 1]
```diff
@@ -0,0 +1,520 @@
+/**
```
Comment (vovka) [inline vendor/assets/javascripts/dirPagination.js]: Move this file to `/vendors`

**Comment by YUzhva** [comment]
What about to change location of the third party modules from other developers like dirPagination from
/javascripts/dirPagination.js
to
/javascripts/angularjs-app/**modules**/dirPagination.js (instead of modules we could use utils, libraries, lib, etc. - that would be folder for third party extensions)

File: vendor/assets/javascripts/dirPagination.js (hunk 1 above)
Comment (superNazo) [inline vendor/assets/javascripts/dirPagination.js]: it is in vendor/assets/javascripts/angularjs_app/dirPagination.js

**Comment by anatoliiibrahimov** [comment]
seems to be fine :+1:

**Comment by vovka** [comment]
@superNazo What's the status of this task?

**Comment by superNazo** [comment]
@vovka in progress ...

File: app/controllers/quizzes_controller.rb [hunk 2]
```diff
@@ -4,7 +4,13 @@ class QuizzesController < ApplicationController
 
   # GET /quizzes
   def index
-    @quizzes = Quiz.all.paginate(page: params[:page], per_page: 10)
+    @quizzes = Quiz.all.paginate(page: params[:page], per_page: 2)
+
+
```
Comment (vovka) [inline app/controllers/quizzes_controller.rb]: Codacy Issue found: [Extra blank line detected.](https://www.codacy.com/app/vovka/video_quiz/file/2500527876/issues/source?bid=3205144&fileBranchId=3237995#l9)

File: app/assets/javascripts/angularjs_app/components/quizzes/quizzesControllers.js [hunk 3]
```diff
@@ -47,6 +77,20 @@ quizzesControllers.controller('indexQuizCtrl',
   }
 ]);
 
+quizzesControllers.controller('paginationCtrl',
+  ['$scope', 'paginationService',
+  function($scope, paginationService) {
+    $scope.quizzes = $scope.quizzes_array;
```
Comment (vovka) [inline app/assets/javascripts/angularjs_app/components/quizzes/quizzesControllers.js]: Codacy Issue found: [Identifier 'quizzes_array' is not in camel case.](https://www.codacy.com/app/vovka/video_quiz/file/2500527924/issues/source?bid=3205144&fileBranchId=3237995#l83)

Comment (vovka) [inline app/assets/javascripts/angularjs_app/components/quizzes/quizzesControllers.js]: Codacy Issue found: [Identifier 'quizzes_array' is not in camel case.](https://www.codacy.com/app/vovka/video_quiz/file/2500544524/issues/source?bid=3205144&fileBranchId=3237995#l83)

File: app/controllers/quizzes_controller.rb (hunk 2 above)
Comment (vovka) [inline app/controllers/quizzes_controller.rb]: Codacy Issue found: [Extra blank line detected.](https://www.codacy.com/app/vovka/video_quiz/file/2500544476/issues/source?bid=3205144&fileBranchId=3237995#l9)

**Pull request closed without merging**